app/javascript/packs/stylesheets/patient.scss

```diff
@@ -0,0 +1,3 @@
+.red-icon {
+  color: red;
```

Comment:
Any chance we can change this to a slightly flatter version of red? The `red` color is a little vibrant for our color-scheming. Maybe `#CE2717` ?

`#CE2717` is on the left, `red` on the right

<img width="522" alt="image" src="https://user-images.githubusercontent.com/17532163/111470990-cb909f00-86fe-11eb-84a1-b007b7a2bd14.png">

Reply:
Also @hackrm , the name should be changed. Maybe `.symptomatic-icon`?

Reply:
sure! I'll make that adjustment with the next commit

Reply:
should we run these options with the PO team

Reply:
I'll ask them quickly & will follow up here with their response

Reply:
PO team prefers the brighter shade of red, will revert back to that color in my next commit

Reply:
I guess the Users preferred the brighter red, so ignore my comments! Well not the class-renaming one, but the color one.

Reply:
Sounds good! Good idea @timwongj to run it by them.